src/style.css

```diff
@@ -0,0 +1,132 @@
+/* General settings*/
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  /* margin: 0; */
+  --search-icon: url("./images/Search.svg");
+}
+/* End General settings*/
+
+/* .search-bar {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 0 10px;
+} */
+
+.search-bar__form {
+  margin-top: 20px;
+  display: flex;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+/* Input--big settings*/
+.search-bar__icon--big {
+  background-image: var(--search-icon);
+  background-size: 19px;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+}
+
+.search-bar__form--main {
+  height: 68px;
+  position: relative;
+}
+
+.search-bar__input--big {
+  margin: 25px auto 26px 62px;
+  width: 100%;
+  border: none;
+  outline: none;
+}
+
+.search-bar__input--big::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  color: #3d4e61;
+}
+
+.search-bar__input--big:focus {
+  text-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  font-family: "Avenir-Heavy", sans-serif;
+}
+
+.search-bar__input--main-input:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-bar__input--main-input:hover {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+/* End Input--big settings*/
+
+/* Input--small settings*/
+.search-bar__form--secondary {
+  height: 40px;
+  position: relative;
+}
+
+.search-bar__input--small {
+  margin: 12px auto 11px 33px;
+  width: 100%;
+  border: none;
+  outline: none;
+  text-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__icon--small {
+  background-image: var(--search-icon);
+  background-size: 11px;
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  bottom: 16px;
+}
+
+.search-bar__input--small::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 14px;
+  font-weight: 400;
```

Comment:
400 is a default font-weight, so it's redundant here

Reply:
not fixed